tidb-cloud/cli-reference.md

```diff
@@ -0,0 +1,111 @@
+---
+
+# TiDB Cloud CLI Reference
+
+TiDB Cloud CLI is a command line interface, which allows you to operate TiDB Cloud from your terminal with a few lines of commands. In the TiDB Cloud CLI, you can easily manage your TiDB Cloud clusters, import data to your clusters, and perform more operations.
+
+## Before you begin
+
+Make sure to first [set up your TiDB Cloud CLI environment](/tidb-cloud/get-started-with-cli.md). Once you have installed the `ticloud` CLI, you can use it to manage your TiDB Cloud clusters from the command lines.
+
+## Available commands
+
+The following table lists the available commands for the TiDB Cloud CLI.
+
+To use the `ticloud` CLI in your terminal, you can use `ticloud [command] [subcommand]`. For users using [TiUP](https://docs.pingcap.com/tidb/stable/tiup-overview), use `tiup cloud [command] [subcommand]` instead.
+
+| Command    | Subcommands                                                | Description                                                                                              |
+|------------|------------------------------------------------------------|----------------------------------------------------------------------------------------------------------|
+| cluster    | create, delete, describe, list                             | Manage clusters                                                                                          |
+| completion | bash, fish, powershell, zsh                                | Generate completion script for specified shell                                                           |
+| config     | create, delete, describe, edit, list, set, use             | Configure user profiles.                                                                    |
+| help       | cluster, completion, config, help, import, project, update | View help for any command                                                                                |
+| import     | cancel, describe, list, start                              | Manage [import](/tidb-cloud/tidb-cloud-migration-overview.md#import-data-from-files-to-tidb-cloud) tasks |
+| project    | list                                                       | Manage projects                                                                                          |
+| update     |                                                            | Update the CLI to the latest version                                                                     |
+
+## Command modes
+
+The TiDB Cloud CLI provides the following two modes for some commands for easy use:
+
+- Interactive mode
+
+    In this mode, you can run a command without flags (such as `ticloud config create`), and the CLI prompts you for input.
+
+- Non-interactive mode
+
+    In this mode, you must provide all the required arguments and flags when running a command, such as `ticloud config create --profile-name <profile-name> --public-key <public-key> --private-key <private-key>`
```

Comment:
According to https://developers.google.cn/style/word-list#letter-s, generally avoid using "should",  because should is ambiguous by definition, and it can be problematic. For example, if you're telling the reader what to do, "should" implies that the action is recommended but optional, leaving the reader unsure about what to do." 

While, "must" is used to describe a required action or state (for example, You must have the Editor role)."  Therefore, either "must" or "need to" works better than "should".